swarm-derive/src/lib.rs

```diff
@@ -645,3 +613,15 @@ fn is_ignored(field: &syn::Field) -> bool {
 
     false
 }
+
+fn capitalize(ident: Ident) -> Ident {
+    let name = ident.to_string();
+    let mut chars = name.chars();
+    let capitalized = chars
+        .next()
+        .expect("Identifier to have at least one character.")
+        .to_uppercase()
+        .collect::<String>()
+        + chars.as_str();
+    Ident::new(&capitalized, Span::call_site().into())
+}
```

Comment:
We use capitalize for the `Ident`s of our data struct fields, right? Since those are written in `snake_case`, shouldn't we iter through all chars, filter out "`_`" and capitalize the subsequent char? Otherwise we might end up with something like `Gossipsub_behaviourEvent`.

Reply:
:facepalm: Oh, good catch. Silly me.